main.go

```diff
@@ -203,6 +203,8 @@ func main() {
 		log.Info("Using default AutoscalingListener logging parameters", "LogLevel", actionsgithubcom.DefaultScaleSetListenerLogLevel, "LogFormat", actionsgithubcom.DefaultScaleSetListenerLogFormat)
 	}
 
+	actionsgithubcom.SetListenerEntrypoint(os.Getenv("LISTENER_ENTRYPOINT"))
```

Comment:
Since we're feature flagging this, let's pass which listener we are using in the UserAgent so that we can troubleshoot problems pre-emptively and more effectively

Reply:
I approved the other PR, let's also log the type of listener we're using